src/displayapp/screens/ApplicationList.h

```diff
@@ -36,9 +36,10 @@ namespace Pinetime {
         static constexpr int appsPerScreen = 6;
 
         // Increment this when more space is needed
-        static constexpr int nScreens = 2;
+        static constexpr int nScreens = 3;
 
         static constexpr std::array<Tile::Applications, appsPerScreen * nScreens> applications {{
+          {Symbols::gallery, Apps::Gallery},
```

Comment:
Maybe don't put this in the first slot. I suggest the slot of the Motion app which in turn gets pushed to the end of the list.

Reply:
Sorry, I did this mostly for testing actually. I'm happy to put it where you suggest so that it's
```cpp
          {Symbols::stopWatch, Apps::StopWatch},
          {Symbols::clock, Apps::Alarm},
          {Symbols::hourGlass, Apps::Timer},
          {Symbols::shoe, Apps::Steps},
          {Symbols::heartBeat, Apps::HeartRate},
          {Symbols::music, Apps::Music},

          {Symbols::paintbrush, Apps::Paint},
          {Symbols::paddle, Apps::Paddle},
          {"2", Apps::Twos},
          {Symbols::gallery, Apps::Gallery},
          {Symbols::drum, Apps::Metronome},
          {Symbols::map, Apps::Navigation},
          {Symbols::chartLine, Apps::Motion},
```
Fixed this with 2bef804b